.pre-commit-config.yaml

```
@@ -6,30 +6,39 @@ repos:
       rev: v6.0.0
       hooks:
           - id: check-added-large-files
-            args: ["--maxkb=10240"]
+            args: [ "--maxkb=10240" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)